src/firestore-query.js

```diff
@@ -190,3 +189,5 @@ function extractName(path) {
 }
 
 module.exports = MockFirestoreQuery;
+Octotree is enabled on this page. Click this button or press cmd shift s (or ctrl shift s) to show it.
```

Comment:
???

Reply:
Apologize for my misstake.

Reply:
No problem! Just didn't want that to slip through :)